a10_octavia/controller/worker/tasks/virtual_port_tasks.py

```diff
@@ -28,6 +29,18 @@
 
 class ListenersParent(object):
 
+    def transform_https_protocols(self, listener):
```

Comment:
We should just be able to add listener mappings to here: https://github.com/a10networks/a10-octavia/blob/master/a10_octavia/common/openstack_mappings.py#L16

We can make a mapping call like: https://github.com/a10networks/a10-octavia/blob/b71cb06c142eda10c22e4d9a9acd5adda38b20af/a10_octavia/controller/worker/tasks/health_monitor_tasks.py#L105

Reply:
Done.